tools/add_triage_signature.py

```diff
@@ -582,14 +564,81 @@ def isVMware(host):
             self._update_triaging_ticket(issue_key, report, should_update=should_update)
 
 
+class AgentStepFailureSignature(Signature):
+    """
+    This signature creates a report of all agent step failures that were encountered in the logs
+    """
+    PATTERN = re.compile(
```

Comment:
Can you set a link to the place this log happens? It seems that we will have to be consistent with it